dev/com.ibm.ws.microprofile.openapi/resources/com/ibm/ws/microprofile/openapi/impl/validation/resources/ValidationMessages.nlsprops

```diff
@@ -32,6 +32,11 @@ infoTermsOfServiceInvalidURL=The Info object must contain a valid URL. The \"{0}
 invalidUrl=The \"{0}\" object must contain a valid URL. The \"{1}\" value specified for the URL is not valid.
 invalidUri=The \"{0}\" object must contain a valid URI. The \"{1}\" value specified for the URI is not valid.
 
+headerExampleOrExamples=The Header object \"{0}\" specifies both an example object and an examples object. Only one of them should be specified.
+headerSchemaAndContent=The Header object \"{0}\" MUST contain either a schema property, or a content property, but not both.
+headerSchemaOrContent=The Header object \"{0}\" MUST contain either a schema property or a content property.
+headerContentMap=The "content" map within header object \"{0}\" must only contain one entry.
```

Comment:
Please escape the quote with slash (\\) in front of them.
"content" --> \\"content\\"

FYI: I added the slash in previous comment, but GitHub when showing the comment removed/escaped it, so this time I added an escape for the slash [with a slash :)]